package/network/services/odhcpd/Makefile

```diff
@@ -32,7 +32,7 @@ define Package/odhcpd/default
   SECTION:=net
   CATEGORY:=Network
   TITLE:=OpenWrt DHCPv6(-PD)/RA Server & Relay
-  DEPENDS:=+libubox +libuci +libubus +libnl-tiny
+  DEPENDS:=+libubox +libuci +libubus +libnl-tiny @IPV6
```

Comment:
NAK odhcpd has DHCPv4 support so it cannot solely depend on the config symbol IPv6